src/main/java/com/easypost/model/CarrierAccount.java

```diff
@@ -58,11 +58,19 @@ public CarrierAccount update(Map<String, Object> params, String apiKey) throws E
 
 		// delete
 		public void delete() throws EasyPostException {
-			this.delete(null);
+			this.delete(null, null);
 		}
-		public void delete(String apiKey) throws EasyPostException {
+		public void delete(String apiKey, String placeholder) throws EasyPostException {
```

Comment:
Is this placeholder string necessary for something? Same with below, I feel like I'm missing the point